rules/windows/create_stream_hash/create_stream_hash_susp_ip_domains.yml

```diff
@@ -0,0 +1,38 @@
+title: Unusual File Download from Direct IP Address
+id: 025bd229-fd1f-4fdb-97ab-20006e1a5368
+status: experimental
+description: Detects the download of suspicious file type from URLs with IP
+author: Nasreddine Bencherchali
+references:
+    - https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=90015
```

Comment:
As the ref is not the artefact : 
https://github.com/trustedsec/SysmonCommunityGuide/blob/adcdfee20999f422b974c8d4149bf4c361237db7/chapters/file-stream-creation-hash.md